nixos/infrastructure/flyingcircus-virtual.nix

```diff
@@ -4,6 +4,9 @@ with lib;
 
 let
   cfg = config.flyingcircus;
+  ioScheduler =  if ((cfg.infrastructure.preferNoneSchedulerOnSsd == true) && (cfg.enc.parameters.rbd_pool == "rbd.ssd"))
```

Comment:
cfg.infrastructure.preferNoneSchedulerOnSsd can be used directly as boolean value, no comparison with true needed